OWNERS_ALIASES

```diff
@@ -480,3 +480,11 @@ aliases:
     - bgrant0607
     - smarterclayton
     - johnbelamaric
+
+  sig-scalability-approvers:
```

Comment:
please sort alphabetically (above sig-scheduling)

Reply:
Done, though entries in this file are not sorted alphabetically, e.g. sig-apps-approvers comes after sig-network-approvers.